File: mifosng-android/src/main/java/com/mifos/objects/accounts/savings/SavingsAccountTransactionResponse.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */

package com.mifos.objects.accounts.savings;

import com.google.gson.annotations.Expose;
import com.mifos.objects.Changes;

/**
 * Created by ishankhanna on 12/06/14.
 */
public class SavingsAccountTransactionResponse {

    @Expose
    private Integer officeId;
    @Expose
    private Integer clientId;
    @Expose
    private Integer savingsId;
    @Expose
    private Integer resourceId;
    @Expose
    private Changes changes;

    public Integer getOfficeId() {
        return officeId;
    }

    public void setOfficeId(Integer officeId) {
        this.officeId = officeId;
    }

    public Integer getClientId() {
        return clientId;
    }

    public void setClientId(Integer clientId) {
        this.clientId = clientId;
    }

    public Integer getSavingsId() {
        return savingsId;
    }

    public void setSavingsId(Integer savingsId) {
        this.savingsId = savingsId;
    }

    public Integer getResourceId() {
        return resourceId;
    }

    public void setResourceId(Integer resourceId) {
        this.resourceId = resourceId;
    }

    public Changes getChanges() {
        return changes;
    }

    public void setChanges(Changes changes) {
        this.changes = changes;
    }
}

File: core/datastore/src/main/java/com/mifos/core/objects/accounts/savings/SavingsAccountTransactionResponse.kt
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.objects.accounts.savings

import com.google.gson.annotations.Expose
import com.mifos.core.objects.Changes
import kotlinx.parcelize.Parcelize

/**
 * Created by ishankhanna on 12/06/14.
 */
data class SavingsAccountTransactionResponse (
    @Expose
    var officeId: Int? = null,

    @Expose
    var clientId: Int? = null,

    @Expose
    var savingsId: Int? = null,

    @Expose
    var resourceId: Int? = null,

    @Expose
    var changes: com.mifos.core.objects.Changes? = null
)